src/controllers/user.js

```diff
@@ -136,3 +136,27 @@ export const updateProfile = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to update new user')
   }
 }
+
+export const createNote = async (req, res) => {
+  const teacherId = req.user.id
+  const studentId = Number(req.params.id)
+  const { content, isEditted } = req.body
+  const newNoteData = {
+    teacherId,
+    studentId,
+    content,
+    isEditted
+  }
+
+  try {
+    if (!content) {
```

Comment:
you're checking if content is provided

However, you should also check if req.user has permissions to create the note. How do you know the user is a TEACHER? you might be able to access this info via `req.user.type`